app/models/vita_partner.rb

```diff
@@ -45,15 +45,21 @@ class VitaPartner < ApplicationRecord
   has_many :child_sites, -> { order(:id) }, class_name: "VitaPartner", foreign_key: "parent_organization_id"
 
   default_scope { includes(:child_sites).order(name: :asc) }
-
+  scope :with_capacity, lambda {
+    capacities = OrganizationCapacity.arel_table
+    joins(:organization_capacity).where(
+      capacities[:active_client_count].lteq(arel_table[:capacity_limit])
+          .or(arel_table[:capacity_limit].eq(nil)) # if capacity_limit is not set, assume org capacity
+    ).where.not(capacity_limit: 0)
```

Comment:
What does it mean for an organization when they have a capacity limit of nil vs 0?